src/frontend/magnify/src/services/array/array.helper.ts

```diff
@@ -0,0 +1,18 @@
+export class ArrayHelper {
+  public static findElementIndex<T>(array: T[], element: T): number | null {
+    const index = array.findIndex((arrayElement: T) => {
+      return arrayElement == element;
+    });
+    return index >= 0 ? index : null;
+  }
+
+  public static removeItem<T>(array: T[], element: T): T[] {
```

Comment:
Why do you not use simple functions here ? 
From my point of view, it allows developer to import only the method they need.

```diff
- import ArrayHelper from './services/array/array.helper`;
+ import { removeItem } from './services/array/array.helper`;
```